src/DAL/migration/v2.2.0.sql

```diff
@@ -0,0 +1,4 @@
+ALTER TABLE "records" ALTER identifier DROP DEFAULT;
```

Comment:
this should be major as it is breaking change

Reply:
fixed